tools/downloader/topologies_test.py

```diff
@@ -73,7 +72,7 @@ def test_result_model(self):
         image_path = os.path.join(os.path.split(os.path.dirname(os.path.realpath(__file__)))[0],
                                   'accuracy_checker', 'data', 'test_data', '1.jpg')
 
-        ie = IECore()
+        ie = inference_engine.IECore()
```

Comment:
Please use `from openvino.inference_engine import IECore` - it's convenient way.